### DISCUSSION TITLE
What did motivate the choice of the currently supported formats ?

### DISCUSSION THREAD

**Discussion by systemasis** [zotify-dev/zotify#18] [General]
I was wondering how did you choose the supported formats and if it'd be possible to add more ?

I took a glance at the code, which seems pretty clean and clear to me considering I'm not much versed in Python, and thought it'd be quite easy to add more formats.

I'd gladly try and put a couple more but I was wondering if you encountered any difficulty in the past with some formats.

**Comment by zotify-dev** [comment]
Simple, they're the one's supported by both FFmpeg and [music-tag](https://zotify.xyz/zotify/music-tag) (tagging library we use).

**Reply by systemasis** [reply]
I see, so it should be possible to add FLAC as a supported format since it's supported by both the aforementioned libraries, right ?

**Comment by AetherVoyager** [comment]
+1

**Reply by AetherVoyager** [reply]
Love to see FLAC too
